.gitignore

```diff
@@ -0,0 +1,2 @@
+build/
```

Comment:
Remove this file?

Reply:
Its actually better to keep this gitignore. It helps a lot in development as you dont have to remove the build befire commiting your changes.

Reply:
Doing `git add -A` is not a good practice. You can achieve similar results with performing `git commit -am "Message"`

Reply:
Okay Ill remove that gitignore file